extraDevfileEntries.yaml

```diff
@@ -15,9 +17,11 @@ samples:
     displayName: Basic Quarkus
     description: A simple Hello World Java application using Quarkus
     icon: https://design.jboss.org/quarkus/logo/final/SVG/quarkus_icon_rgb_default.svg
-    tags: ['Java', 'Quarkus']
-    projectType: quarkus
-    language: java
+    tags:
+      - Java
+      - Quarkus
+    projectType: Quarkus
```

Comment:
is the type not still lower case `quarkus`?

Reply:
`Language` and `projectType` should follow the same case as the commercial display name.